x-pack/plugins/security_solution/server/endpoint/routes/metadata/support/unenroll.ts

```diff
@@ -18,7 +18,8 @@ export async function findAllUnenrolledAgentIds(
       page: pageNum,
       perPage: pageSize,
       showInactive: true,
-      kuery: 'fleet-agents.packages:endpoint AND fleet-agents.active:false',
+      kuery:
+        '(fleet-agents.packages : "endpoint" AND fleet-agents.active : false) OR (NOT fleet-agents.packages : "endpoint" AND fleet-agents.active : true)',
```

Comment:
I can never remember if this is called inverse, converse, contrapositive.

But this seems to include all things running agent, that do not have endpoint configured. Whether that's deleted or not initially configured in the first place. That's fine here, yes?

Reply:
yea, this essentially finds all Agents that:
1. Have been unenrolled while Endpoint is running which is the first case - the record will still think it has endpoint
2. OR when Agent continues running, but we just delete the Endpoint policy which will cause the Agent to uninstall the Endpoint - this the second case

What this is list does is finds all of the ids that we should strike from the metadata list as "unenrolled".

There's probably a more efficient way to do this - I'll check with Franklin when he's back from PTO.